judoscale-rails/lib/judoscale/rails/config.rb

```diff
@@ -3,12 +3,15 @@
 module Judoscale
   module Rails
     module Config
-      attr_accessor :start_reporter_after_initialize, :rake_task_ignore_regex
+      attr_accessor :start_reporter_after_initialize, :rake_task_ignore_regex, :utilization_enabled, :utilization_interval
 
       def reset
         super
         @start_reporter_after_initialize = true
         @rake_task_ignore_regex = /assets:|db:/
+
+        @utilization_enabled = ENV["JUDOSCALE_UTILIZATION_ENABLED"] == "true"
+        @utilization_interval = (ENV["JUDOSCALE_UTILIZATION_INTERVAL"] || 1.0).to_f
```

Comment:
I thought of switching this interval to be in seconds, to match the report interval... also maybe we don't need to be as granular, every ~1 second is enough of a default to track/push metrics?

Reply:
I'm skeptical that we'd lose too much data with 1 second, but we can play with it.

Totally good with the config in seconds vs milliseconds, though.

Reply:
@adamlogic now that this is in GA, can we remove this check?

Reply:
(er, the enabled check, not the interval check)

Reply:
Not quite GA yet... we still need to add this to our Node and Python adapters. We'll remove this check soon.